### ISSUE TITLE
Scrolling on iPad with mouse not working

### ISSUE THREAD

**Issue by ruebenacker** [ajaxorg/ace#4313]
Scrolling in the Editor with mouse is not working on iPad with a mouse or trackpad.
Scrolling with touch gestures is working as expected.
Steps to reproduce: 
iPad with iOS 13.5.1 with mouse / trackpad connected. Go to https://ace.c9.io and enter enough lines in the editor that you can use scrolling. Scroll with trackpad or mouse.

**Comment by nightwing** [comment]
Is mouse wheel support new in 13.5? I have access only to 13.1 and there wheel doesn't work at all. 

If yes could you please check if there are wheel events shown in https://rbyers.github.io/eventTest.html when scrolling over the white region at the bottom right of the page.

**Comment by ruebenacker** [comment]
Yes. Apple rolled out full mouse and trackpad support with iPadOS 13.4 in March. When scrolling i get the output shown in the picture
![4F77B130-1F2E-48AB-9D09-C146DDE5B491](https://user-images.githubusercontent.com/16401415/84074118-53a88200-a9d2-11ea-9bae-0484971b27cd.jpeg)

**Comment by nightwing** [comment]
So ipad instead of sending normal wheel events, changes the scroll position directly :(

There is a bug  report about this at https://bugs.webkit.org/show_bug.cgi?id=210071

If they do not fix the issue quickly, it may be possible to find some hacky workaround, e.g. make scrollbar div fill the whole editor area, but that can cause other problems. I'll try to find a workaround once i get access to a simulator for ipad 13.4.

**Comment by hexise** [comment]
It seems the bug https://bugs.webkit.org/show_bug.cgi?id=210071 has been fixed. Is there any update for this issue from Ace Editor part?

**Comment by hexise** [comment]
@andrewnester yes, I can reproduce this problem on my iPad air 4 with iOS 15 and latest ace build. I connected to iPad using my logi mouse with bluetooth. You can drag and move the scrollbar using mouse, but the mouse wheel did not work, nothing happens when I spins mouse wheel.

**Comment by hexise** [comment]
I want to keep this issue open.

**Comment by chazeon** [comment]
I want to keep this issue open.

**Comment by hexise** [comment]
I want to keep this issue open.

**Comment by hexise** [comment]
I want to keep this issue open.
